playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -61,6 +63,23 @@
     when:
     - _default_registry.results.results[0] != {}
 
+  - name: Check for registry-console
+    oc_obj:
+      state: list
+      kind: dc
+      name: registry-console
+    register: _registry_console
+
+  - name: Update registry-console image to current version
+    oc_edit:
+      kind: dc
+      name: registry-console
+      namespace: default
+      content:
+        spec.template.spec.containers[0].image: "{{ registry_console_image }}"
+    when:
+    - _registry_console.results.results[0] != {}
+
```

Comment:
Since the origin image is always latest (`image: cockpit/kubernetes:latest`) we won't have to update the dc when `openshift.common.deployment_type != 'origin'`.